fullstop-web/fullstop-violations-rest/src/test/java/org/zalando/stups/fullstop/web/controller/ViolationsControllerTest.java

```diff
@@ -79,6 +86,18 @@
     public void setUp() throws Exception {
         reset(violationServiceMock, mockTeamOperations, mockViolationConverter);
 
+        createViolation = new CreateViolation();
+        createViolation.setEventId(UUID.randomUUID().toString());
+        createViolation.setAccountId("2547093960");
+        createViolation.setRegion("eu-west-1");
+        createViolation.setMetaInfo("this is a meta info object of string");
```

Comment:
please test with meta info of type Map<String,String> as this is more common use case